webapp/src/containers/components/modals/actionModal.tsx

```diff
@@ -1,80 +1,79 @@
-import React from 'react';
 
-import Button from 'react-bootstrap/Button';
-import Modal from 'react-bootstrap/Modal';
-import Table from 'react-bootstrap/Table';
-import Form from 'react-bootstrap/Form';
-
-import './styles.css';
+import {Configs} from 'types/plugin/common';
 
 type Props = {
-    visibility: boolean;
-    setVisibility: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
-const ViewActionsModal = ({visibility, setVisibility}: Props) => {
-    const handleClose = () => {
-        setVisibility(false);
-    };
+function ActionModal({visible, setVisible, config, configIndex}: Props) {
+    const actionsLength = config[configIndex]?.actions?.length ?? 0;
+    const attachmentMessageLength = config[configIndex]?.attachmentMessage?.length ?? 0;
```

Comment:
```suggestion
    const actionsLength = config[configIndex]?.actions?.length;
    const attachmentMessageLength = config[configIndex]?.attachmentMessage?.length;
```
I think this will work